terms/conceptdrift_mf.bib

```diff
@@ -0,0 +1,64 @@
+@techreport{Tsymbal04theproblem,
+author = {Tsymbal, Alexey},
+title = {{The Problem of Concept Drift: Definitions and Related Work}},
+year = {2004}
+}
+@book{Bishop:2006:PRM:1162264,
+address = {Berlin, Heidelberg},
+author = {Bishop, Christopher M},
+isbn = {0387310738},
+publisher = {Springer-Verlag},
+title = {{Pattern Recognition and Machine Learning (Information Science and Statistics)}},
+year = {2006}
+}
+@article{Webb:2016:CCD:2962863.2962874,
```

Comment:
see guidelines in readme -- point 8.  and clean all bib entries.

Reply:
done